pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "alpaca-py"
-version = "0.11.1"
+version = "0.12.0"
 description = "The Official Python SDK for Alpaca APIs"
 authors = [
     "Rahul Chowdhury <rahul.chowdhury@alpaca.markets>",
```

Comment:
Out of context, may we add more people here?

Reply:
Seems yes. https://python-poetry.org/docs/pyproject/#authors
We can consider to set a group mail address here instead of individual one.